modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc

```diff
@@ -28,29 +28,60 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. From the *Developer* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
+. In the *Developer* perspective of the {product-title} web console, click *Observe* and go to the *Metrics* tab.
 
 . Select the project that you want to view metrics for in the *Project:* list.
 
-. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*. The metrics from the queries are visualized on the plot.
+. To add one or more queries, perform any of the following actions:
++
+|===
+|Option |Description
+
+|Select an existing query.
+|From the *Select query* drop-down list, select an existing query.
+
+|Create a custom query.
+|Add your Prometheus Query Language (PromQL) query to the *Expression* field.
+
+As you type a PromQL expression, autocomplete suggestions appear in a drop-down list. These suggestions include functions, metrics, labels, and time tokens.
+You can use the keyboard arrows to select one of these suggested items and then press Enter to add the item to your expression. You can also move your mouse pointer over a suggested item to view a brief description of that item.
+
+|Add multiple queries. |Click *Add query*.
+
+|Duplicate an existing query. |Click the Options menu {kebab} next to the query, then choose *Duplicate query*.
+
+|Disable a query from being run. |Click the Options menu {kebab} next to the query and choose *Disable query*.
+|===
+
+. To run queries that you created, click *Run queries*. The metrics from the queries are visualized on the plot. If a query is invalid, the UI shows an error message.
 +
 [NOTE]
 ====
-In the Developer perspective, you can only run one query at a time.
+* Queries that operate on large amounts of data might time out or overload the browser when drawing time series graphs. To avoid this, click *Hide graph* and calibrate your query by using only the metrics table. Then, after finding a feasible query, enable the plot to draw the graphs.
```

Comment:
"When drawing time series graphs, queries that operate on large amounts of data might time out or overload the browser."